### DISCUSSION TITLE
TagInput New Line Paste Separator

### DISCUSSION THREAD

**Discussion by photogsy** [buefy/buefy#3981] [Ideas]
### Description

It would be super useful to be able to paste multiple lines into the tag input and have these split into individual tags. I tried to add `'\n'` and `'\r'` in the `on-paste-separators` prop but this doesn't work.

### Why Buefy need this feature

We often have situations where we need to add lots of tags into the tag input which contain commas, semicolons and other characters, so new line is really the only way to separate them for easy copy paste.

**Comment by wesdevpro** [comment]
I do like this idea. However, I will leave most feature requests up to @jtommy to approve then I will work on them. Because my goal is just to maintain and fix bugs for Buefy v2 and migrate the project to Buefy v3. Thank you for your feature request
